docs/INSTALLATION.md

```diff
@@ -2,11 +2,18 @@
 
 ### Table of Contents
 
+* [Notice on non amd64](#non-amd64)
```

Comment:
minor; can you please put this link below `Docker Image` title?

Reply:
here you are